### PR TITLE
Increase scope of use

### PR DESCRIPTION
**Pull request by GPKyte**
Added new tables to track users and channels.
Added export feature to create Slack-ish export
Changed naming convention
Improved error-handling messages and comments

### CODE REVIEW

File: archivebot.py [hunk 1]
```diff
@@ -1,178 +1,211 @@
...
+    ENV['id_user'] = dict([(m['id'], m['name']) for m in info['members']])
+
+    args = []
+    for m in info['members']:
+        args.append((
+            m['name'],
+            m['id'],
+            m['profile'].get('image_32', 'https://secure.gravatar.com/avatar/c3a07fba0c4787b0ef1d417838eae9c5.jpg?s=32&d=https%3A%2F%2Ffst.slack-edge.com%2F66f9%2Fimg%2Favatars%2Fava_0024-32.png')
+        ))
+    cursor.executemany("INSERT INTO users(name, id, avatar) VALUES(?,?,?)", args)
```
Comment (docmarionum1) [inline archivebot.py]: I get an error here due to the create table command having `text avatar` instead of `avatar text`.

Comment (GPKyte) [reply archivebot.py]: Forgot that I changed this on my branch but not on this pull, thanks for pointing it out. I pushed the updates.

**Comment by docmarionum1** [comment]
Thanks @GPKyte !  I made a couple of changes to `export.py` to make it python3 compatible.  Do you want to double check and make sure it still works for you?

**Comment by GPKyte** [comment]
This looks good to me! Thanks @docmarionum1

**Pull request merged**